src/pages/SwapNew/components/SwapInfo/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { Trade } from '@swapr/sdk'
+
+import { AnimatePresence, motion } from 'framer-motion'
+import { useState } from 'react'
+import styled from 'styled-components'
+
+import {
+  ELEMENTS_BACKGROUND_SECONDARY,
+  ELEMENTS_BORDER_SECONDARY,
+  ELEMENTS_SPACING,
+  BorderStyle,
+} from '../../constants'
+import { SwapInfoBasics } from './SwapInfoBasics'
+import { SwapInfoDetailed } from './SwapInfoDetailed'
+
+type SwapInfoProps = {
+  loading: boolean
+  allPlatformTrades?: (Trade | undefined)[]
+  selectedTrade?: Trade
+  outputCurrencySymbol?: string
+}
+
+export function SwapInfo({ loading, allPlatformTrades, selectedTrade, outputCurrencySymbol }: SwapInfoProps) {
+  const [showSwapInfoDetails, setShowSwapInfoDetails] = useState(false)
+
+  return (
+    <Container
+      initial={{ height: 0 }}
+      animate={{
+        height: 'auto',
+      }}
+    >
```

Comment:
Not sure about this animation. Feels a bit sloppy at least in my computer

Reply:
Animation removed, maybe we can bring it back later.